docs/YAML_documents/yaml_structure.md

```diff
@@ -0,0 +1,31 @@
+YAML parser will receive a test yaml file as an input. 
+
+A test yaml file should have the following parameters:
+
+* test_id
+* object_size
+* part_size
+* result_duration
+* sessions_per_node
+* sessions
+* tool
+
+---
+
```

Comment:
Add explanation for range_read key

Reply:
Please add explanation for every keyword. i.e. session_per_node

Reply:
added